cura/Machines/QualityManager.py

```diff
@@ -340,6 +340,13 @@ def getQualityGroupsForMachineDefinition(self, machine: "GlobalStack") -> dict:
 
         return quality_group_dict
 
+    def getDefaultQualityType(self, machine: "GlobalStack") -> Optional[QualityGroup]:
+        preferred_quality_type = machine.definition.getMetaDataEntry("preferred_quality_type")
+        quality_group_dict = self.getQualityGroups(machine)
+        quality_group = quality_group_dict.get(preferred_quality_type)
```

Comment:
Not sure if tests will complain that if `quality_group_dict` or `preferred_quality_type` are None, this line will not work.